### PR TITLE
Fix test formatting

### PR DESCRIPTION
**Pull request by okwolf**
Updating format command to include test js files and caught one that wasn't formatted.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "bundle": "rollup -i src/index.js -o dist/hyperapp.js -m -f umd -n hyperapp",
     "minify": "uglifyjs dist/hyperapp.js -o dist/hyperapp.js -mc pure_funcs=Object.defineProperty --source-map includeSources,url=hyperapp.js.map",
     "prepare": "npm run build",
-    "format": "prettier --semi false --write \"src/**/*.js\" \"{,test/ts/}*.{ts,tsx}\"",
+    "format": "prettier --semi false --write {src,test}/**/*.js {,test/ts/}*.{ts,tsx}",
```
Comment (JorgeBucaran) [inline package.json]: @okwolf Didn't you made the previous commit yourself to support windows or something?

Comment (okwolf) [reply package.json]: The previous change was from single quotes to double quotes, but it doesn't look like quotes are even needed at all.

Comment (JorgeBucaran) [reply package.json]: @okwolf Sure, not needed in unix.

Comment (okwolf) [reply package.json]: @JorgeBucaran verified still working on Windows too! 🎉

**Pull request merged**